build.gradle

```diff
@@ -57,22 +63,28 @@ dependencyManagement {
 
 dependencies {
 
-	annotationProcessor 'org.projectlombok:lombok'
+    annotationProcessor 'org.projectlombok:lombok'
+
+    compileOnly 'org.projectlombok:lombok'
 
-	compileOnly 'org.projectlombok:lombok'
+    implementation 'org.springframework.boot:spring-boot-starter'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
 
-	implementation 'org.springframework.boot:spring-boot-starter'
-	implementation 'org.springframework.boot:spring-boot-starter-web'
-	implementation 'net.logstash.logback:logstash-logback-encoder:6.2'
+    implementation 'net.logstash.logback:logstash-logback-encoder:6.2'
+    implementation 'com.microsoft.azure:applicationinsights-spring-boot-starter:2.5.1'
+    implementation 'com.microsoft.azure:applicationinsights-logging-logback:2.5.1'
+    agentDeps 'com.microsoft.azure:applicationinsights-agent:2.5.1'
+    implementation configurations.agentDeps
 
-	implementation 'software.amazon.awssdk:aws-sdk-java:2.10.0'
-	implementation 'software.amazon.awssdk:sns:2.10.0'
+    implementation 'software.amazon.awssdk:aws-sdk-java:2.10.0'
+    implementation 'software.amazon.awssdk:sns:2.10.0'
 
-	testCompile("org.assertj:assertj-core:3.6.1")
-	testImplementation('org.junit.jupiter:junit-jupiter-api:5.4.2')
-	testRuntime('org.junit.jupiter:junit-jupiter-engine:5.4.2')
+    implementation 'org.apache.commons:commons-lang3:3.9'
 
-	testCompile("org.springframework.boot:spring-boot-starter-test:2.2.1.RELEASE")
+    testRuntime('org.junit.jupiter:junit-jupiter-engine:5.4.2')
+    testImplementation('org.junit.jupiter:junit-jupiter-api:5.4.2')
+    testImplementation("org.assertj:assertj-core:3.6.1")
```

Comment:
worth also getting into the habit of running `./gradlew dependencyUpdates` before you check in and updating the project to match (including gradle version).  Latest version of assertj is `3.14.0`